app/upgrades/v1/mainnet/params_update.go

```diff
@@ -0,0 +1,123 @@
+package mainnet
+
+import (
+	"fmt"
+	"time"
+
+	"cosmossdk.io/math"
+	sdkmath "cosmossdk.io/math"
+	"github.com/babylonlabs-io/babylon/app/keepers"
+	appparams "github.com/babylonlabs-io/babylon/app/params"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
+	// Slashing params
+	MainnetMinSignedBlocks, _    = sdkmath.LegacyNewDecFromStr("0.6")
+	MainnetSlashFractionDowntime = sdkmath.LegacyZeroDec()
+	MainnetDowntimeJailDuration  = 300 * time.Second
+
+	// Governance params
+	MainnetVotingPeriod = 3 * 24 * time.Hour
+	// 50k BBN
+	MainnetMinDeposit = sdk.NewCoin(appparams.DefaultBondDenom, math.NewInt(50_000_000000))
+	// 200k BBN
```

Comment:
```suggestion
	// 200k BABY
```

Reply:
considering this nomenclature change, should we update also the [`HumanCoinUnit`](https://github.com/babylonlabs-io/babylon/blob/e0a4b1b1e9e29ef73f441bc9252198fde246765a/app/params/config.go#L13) on the registered denoms @vitsalis ?

Reply:
better not, I don't know what's the impact of this.